<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LiveRoomItem</class>
 <widget class="QWidget" name="LiveRoomItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>160</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>160</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>160</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LiveRoomItem</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 0px;
background-color: #ffffff;</string>
  </property>
  <widget class="QLabel" name="m_liveRoomCoverLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>230</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../res/res.qrc">:/img/test.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="m_liveRoomOwnerAvatarLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../res/res.qrc">:/img/avatar.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="m_liveRoomOwnerNameLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>花未眠</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_liveRoomViewerCountLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>观看人数: 100</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../res/res.qrc"/>
 </resources>
 <connections/>
</ui>
